cmd/age-keygen/age-keygen.1

```diff
@@ -0,0 +1,25 @@
+.Dd June 28, 2020
+.Dt AGE-KEYGEN 1
+.Sh NAME
```

Comment:
This may be invalid mdoc.

According to [mdoc(7)](https://man.openbsd.org/mdoc.7#Os~2), `.Os` is the mandatory third macro for any mdoc file. groff_mdoc(7) is less clear about this; the “TITLE MACROS” section only seems to imply it as convention, rather than necessity.

Also affects `age.1`.